pkgs/tools/backup/grab-site/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
```

Comment:
all new packages must be added to by-name hierarchy  

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme